tests/accounts.test.js

```diff
@@ -0,0 +1,760 @@
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Accounts Email", () => {
+        it("GET /Accounts Email", (done) =>{
+            chai.request(server).get("/accounts/email/martha_15_7@outlook.com")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(1);
+                done();
+            });
+        });
+
+        it("GET /Accounts Email Not Found", (done) =>{
+            chai.request(server).get("/accounts/email/martha_15_78@outlook.com")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+
+        it("GET /Accounts Email Bad Request", (done) =>{
+            chai.request(server).get("/accounts/email/123")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Accounts Username", () => {
+        it("GET /Accounts Username", (done) =>{
+            chai.request(server).get("/accounts/username/Karla")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(1);
+                done();
+            });
+        });
+
+        it("GET /Accounts Username Not Found", (done) =>{
+            chai.request(server).get("/accounts/username/Karla123")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+
+        it("GET /Accounts Username Bad Request", (done) =>{
+            chai.request(server).get("/accounts/username/  ")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.be.a("object");
+                response.body.should.have.property("errors");
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Accounts DateCreation", () => {
+        it("GET /Accounts DateCreation", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        it("GET /Accounts DateCreation", (done) => {
```